controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java

```diff
@@ -180,10 +172,18 @@ public void initializeStreamWriters(final EventStreamClientFactory clientFactory
                             DeleteTableEvent deleteEvent = new DeleteTableEvent(scope, kvtName, requestId, UUID.fromString(id));
                             return eventHelper.addIndexAndSubmitTask(deleteEvent,
                                     () -> kvtMetadataStore.setState(scope, kvtName, KVTableState.DELETING, context, executor))
-                                    .thenCompose(x -> eventHelper.checkDone(() -> isDeleted(scope, kvtName, context)))
+                                    .thenCompose(x -> eventHelper.checkDone(() -> isDeleted(scope, kvtName)))
                                     .thenApply(y -> DeleteKVTableStatus.Status.SUCCESS);
                         });
-            }), e -> Exceptions.unwrap(e) instanceof RetryableException, NUM_RETRIES, executor);
```

Comment:
do we want to return a failure status from here? the request may have been submitted and still ongoing in the background. right?

Reply:
If there is no response 2 mins after event was submitted this throws timeout exception.

Reply:
Changed this as discussed.